### PR TITLE
Summary

### PR DESCRIPTION
**Pull request by jburel**
This PR

* adds summary: number of images/studies found when querying for compounds
* adds description
* collapses code cells

cc @pwalczysko

### CODE REVIEW

File: Compound.ipynb [hunk 1]
```diff
@@ -5,9 +5,9 @@
    "metadata": {},
    "source": [
     "## Compound\n",
-    "Retrieve images linked to a given set of compounds and check the concentration used.\n",
-    "A OMERO.figure is programatically generated.\n",
-    "This notebook uses the OMERO.mapr API and the OMERO.web API to retrieve data from IDR."
+    "This notebook explores the usage of set of compounds across IDR studies.\n",
```
Comment (pwalczysko) [inline Compound.ipynb:8]:
```suggestion
    "This notebook explores the metadata and images associated with a set of compounds across all IDR studies.\n",
```

File: Compound.ipynb [hunk 2]
```diff
@@ -5,9 +5,9 @@
...
+    "We aim at finding out the range of concentration used across studies.\n",
```
Comment (pwalczysko) [inline Compound.ipynb:9]:
```suggestion
    "We aim at finding out the range of concentrations used across studies for each compoiund.\n",
```

File: Compound.ipynb [hunk 3]
```diff
@@ -5,9 +5,9 @@
...
+    "We retrieve images associated to the compound and programmatically generated an OMERO.figure that can be viewed in any OMERO.server."
```
Comment (pwalczysko) [inline Compound.ipynb:10]:
```suggestion
    "We retrieve images associated to each of the compound and offer all the other metadata associated with those images as a CSV.  Using a subset of these images, we further programmatically generate an OMERO.figure that can be viewed in any OMERO.server."
```

File: Compound.ipynb [hunk 4]
```diff
@@ -429,6 +447,58 @@
...
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from IPython.display import Markdown as md\n",
+    "md(\"The resuls consists of {} studies and {} images.\".format(len(studies), len(pd.unique(df[[\"Image\"]].values.ravel()))))"
```
Comment (pwalczysko) [inline Compound.ipynb:499]: Suggested formulation:
``The result cosists of X images in Z studies.``
(sorry, do not want to susggest in a commit, as can easily make a mistake in the syntax.

**Comment by pwalczysko** [comment]
Couple of formulations suggestions, otherwise looks good and also works fine as tested.

**Comment by jburel** [comment]
@pwalczysko okay now?

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**